core/src/main/java/io/confluent/rest/RestConfig.java

```diff
@@ -199,6 +199,11 @@ public class RestConfig extends AbstractConfig {
       + " runtime request tags in global stats.";
   protected static final boolean METRICS_GLOBAL_STATS_REQUEST_TAGS_ENABLE_DEFAULT = false;
 
+  public static final String SSL_IS_SPIRE_ENABLED_CONFIG = "ssl.spire.enabled";
```

Comment:
[nit] `SSL_IS_SPIRE_ENABLED` -> `SSL_SPIRE_ENABLED`

Reply:
Yes, fix. thank you